src/plugins/image_embeddable/public/components/image_editor/image_editor_flyout.test.tsx

```diff
@@ -35,12 +35,7 @@ const ImageEditor = (props: Partial<ImageEditorFlyoutProps>) => {
             validateUrl,
           }}
         >
-          <ImageEditorFlyout
-            validateUrl={validateUrl}
-            onCancel={() => {}}
-            onSave={() => {}}
-            {...props}
-          />
+          <ImageEditorFlyout onCancel={() => {}} onSave={() => {}} {...props} />
```

Comment:
Since `validateUrl` is provided as part of the `ImageViewerContext`, we don't need to also send it as a prop to `ImageEditorFlyout` - so, I removed it.